2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -37,8 +37,8 @@ collector {
   # The collector returns a cookie to clients for user identification
   # with the following domain and expiration.
   cookie {
-    enabled = true
-    expiration = {{collectorCookieExpiration}} # e.g. "365 days"
+    enabled = true 
+    expiration = {{collectorCookieExpiration}}
```

Comment:
let's leave this as is since people might not be familiar with hocon's duration format